### DISCUSSION TITLE
Is there a way to automatically use SNAPSHOT/nightly version?

### DISCUSSION THREAD

**Discussion by kpritam** [scalameta/nvim-metals#151] [Q&A] [answered]
Every time I have to go to nvim settings and update metals snapshot version.  Is this how everyone does it or theres any alternative to this?

**Comment by ckipp01** [comment]
Great question. This is something I really want, but have been unable to figure out. Essentially I'd like to have an option where you can just use `latest.snapshot` or something for the `metals_server_version`, but I'm unable to get this to work. I've a feature request for this in https://github.com/scalameta/nvim-metals/issues/122.

Part of the issues is that I also can't have a method that just does this by calling Coursier to get the latest snapshot version. I'm not sure why, but with `coursier complete` for example I'm unable to correctly target the snapshots. The behaviour between bootstrap and complete seem to be a bit different there. I have an issue created for that in https://github.com/coursier/coursier/issues/1700.

**Comment by ckipp01** [comment, marked as answer]
Hey @kpritam, I wanted to follow back up on this. You can now just set your version to `SNAPSHOT` and when you update it will pull in the latest.

**Reply by kpritam** [reply]
That's awesome, thanks!

**Reply by ckipp01** [reply]
Btw, I had to rip this out. It wasn't working as expected as the `maven-metadata.xml` I was pulling from keeps being out of sync, so it's not actually correctly pulling the latest version. I've reopened the issue and will need to try and get it fixed upstream.
